cove/templates/base.html

```diff
@@ -2,7 +2,8 @@
 {% load static %}
 {% load bootstrap3 %}
 {% load i18n %}
-<html>
+{% get_current_language as LANGUAGE_CODE %}
+<html lang="{{ LANGUAGE_CODE }}">
```

Comment:
It looks like `LANGUAGE_CODE`  is also defined later on in this template, do we need to remove that deceleration as it's now in the outter scope to avoid any possible shadowing issue in the future? 

https://github.com/OpenDataServices/lib-cove-web/blob/22e5e225cf0285baba3e7f35d38ebd8accc3a981/cove/templates/base.html#L44